src/s_inter.c

```diff
@@ -74,11 +74,34 @@ that didn't really belong anywhere. */
 #include "pthread.h"
 #endif
 
+#ifdef THREADED_IO
+#include "ringbuffer.h"
+
+typedef struct _fdsend {
+    int fds_fd;
+    t_fdsendrmfn fds_fn;
+    void *fds_ptr;
+} t_fdsend;
+#endif // THREADED_IO
+
+typedef enum {
+    kFdpManagerAnyThread,
```

Comment:
Where is `kFdpManagerAnyThread` used?

Reply:
That's right. I guessed it could be useful one day and so `fdp_select()` is ready for it. I can remove it if you advise so.

Reply:
I don't know. Maybe just add a comment that this is not used at the moment.

Reply:
da3e76a6